lib/ddtrace/contrib/httparty/module_patch.rb

```diff
@@ -0,0 +1,30 @@
+require 'ddtrace/ext/net'
+require 'ddtrace/ext/distributed'
+require 'ddtrace/propagation/http_propagator'
+require 'ddtrace/contrib/rest_client/ext'
+
+module Datadog
+  module Contrib
+    module HTTParty
+      # HTTParty ModulePatch
+      module ModulePatch
+        def self.included(base)
+          base.send(:prepend, ClassMethods)
```

Comment:
Curious; why `prepend` over `include` here?

Reply:
Ah. I was playing around with monkey-patching some methods on the `HTTParty` before deciding against it. Changing this to `:include`.